Simple Incremental/Assets/Scripts/Movement.cs

```diff
@@ -23,6 +23,7 @@ public class Movement : MonoBehaviour
 
     public void Start()
     {
+        target = GameObject.FindWithTag("Player").transform;

```

Comment:
Do not modify other features. Please remove.

This is not needed anyways. Target is serialized and exposed to the inspector. This is configured by the designer.

Reply:
All newly spawned enemies had null targets and would not move.

Reply:
I can remove but the targetting will need to be fixed and i wanted to see the enemies spawning and moving.

Reply:
I can remove but the targetting will need to be fixed and i wanted to see the enemies spawning and moving.

Reply:
I'm guessing your prefab did not have the Target defined in the inspector?

If you update your prefab, newly spawned enemies should have targets.